test/scenarios-periodics/el92-src@offline.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# Sourced from scenario.sh and uses functions defined there.
+
+scenario_create_vms() {
+    prepare_kickstart host1 kickstart-offline.ks.template rhel-9.2-microshift-source-offline
+    # Create a VM with 0 NICs
+    launch_vm host1 rhel-9.2-microshift-source-offline "" "" "" "" 0
```

Comment:
At some point, in a different PR, we should introduce argument parsing for this function.

Reply:
I agree. It looks awful :). 
Opened https://issues.redhat.com/browse/USHIFT-1810 to handle this later.